util/recover_test.go

```diff
@@ -20,7 +20,7 @@ func TestRecoverHandler(t *testing.T) {
 		Convey("without a panic there should be no log messages", func() {
```

Comment:
Let's add test coverage for the methods in recover.go that you added

Reply:
done